Libraries/PeriphDrivers/Source/SPI/spi_me21.c

```diff
@@ -65,6 +65,20 @@ int MXC_SPI_Init(mxc_spi_regs_t *spi, int masterMode, int quadModeUsed, int numS
         MXC_SYS_ClockEnable(MXC_SYS_PERIPH_CLOCK_SPI0);
         MXC_SYS_Reset_Periph(MXC_SYS_RESET0_SPI0);
         MXC_GPIO_Config(&gpio_cfg_spi0);
+
+        //ME21 SPI0 supports ss0-ss3 ship select
+        if (numSlaves & 0x01) {
+            MXC_GPIO_Config(&gpio_cfg_spi0_ss0);
+        }
+        if (numSlaves & 0x02) {
```

Comment:
I think the logic here may be incorrect, since numSlaves is an integer value and not a bit mask.

See [spi_reva.c](https://github.com/Analog-Devices-MSDK/msdk/blob/9a9bfbbf696cd6eec64f78bb98fd00a4ce58eeb3/Libraries/PeriphDrivers/Source/SPI/spi_reva.c#L112)

I think the logic here should be 
```C
if (numSlaves >= 1) {
    MXC_GPIO_Config(&gpio_cfg_spi0_ss0);
}
if (numSlaves >= 2) {
    MXC_GPIO_Config(&gpio_cfg_spi0_ss1);
}
if (numSlaves >= 3) {
    MXC_GPIO_Config(&gpio_cfg_spi0_ss2);
}
if (numSlaves == 4) {
    MXC_GPIO_Config(&gpio_cfg_spi0_ss3);
}
```

Reply:
Fixed as an intermediate solution for now.  Later on, I would like to change all devices' SPI files and will define a bit mask that will allow the user to use any chip select.